README.md

```diff
@@ -86,7 +86,35 @@ This POST creates a returns an `instanceId` which is a new resource, which popul
 1. Run `npm run build`. The build result files are under the build directory. (Note: Change the relative path of api_demo_icon.svg, if it is not deployed to root folder.)
 1. Alternatively, run `npm start` to debug the application.
 
-The below document is generated automatically by Create React App:
+## Configure the Sample
+
+The config.js file is used to configure the sample for your environment. Set values for the following variables to configure the Intelligence Server, project, folder, user credentials, and number of documents to display on the page.
+
+| Variable name | Variable description |
+|:-------------|:------------------|
+| restServerURL | Base URL of the REST API. For example, http://localhost:8080/MicroStrategyLibrary/api. The REST API endpoints will be appended to this base URL. |
+| projectId | ID of the MicroStrategy project that will be used by the REST API endpoints. For example, "B19DEDCC11D4E0EFC000EB9495D0F44F". |
+| username | Username that will be authenticated. |
+| password| Password for username that will be authenticated. |
```

Comment:
`username` and `password` variables seem to be unused. Do we need to remove them here?
```suggestion
```